method_name: "regression"
dataset_top_path: "/home/ssd_dir/dataset_image_to_gravity_ozaki"
experiment_type: "/stick"
image_env: "/1cam"
train: #set sequence
  - "/campus_cww"
  - "/dkan_indoor_without_mocaparea"
  - "/dkan_outdoor"
valid:
  - "/dkan_outdoor_cw"
csv_name: "imu_camera.csv"

weights_path: "../../weights/test1"

hyperparameter:
  resize: 224
  mean_element: 0.5
  std_element: 0.5
  hor_fov_deg: 70
  optimizer_name: "Adam"
  lr_cnn: 1e-5
  lr_fc: 1e-4
  batch_size: 70
  num_epochs: 75
